builtin/logical/pki/storage.go

```diff
@@ -1174,6 +1174,12 @@ func (sc *storageContext) getRevocationConfig() (*crlConfig, error) {
 		result.Version = 1
 	}
 
+	// Depending on client version, it's possible that the expiry is unset.
+	// This sets the default value to prevent issues in downstream code.
+	if result.Expiry == "" {
+		result.Expiry = defaultCrlConfig.Expiry
+	}
```

Comment:
Yep! This is what I was expecting. The remaining parameters shouldn't have empty string values, as they're all new with 1.12.